.env.example

```
@@ -22,6 +22,10 @@ NEXT_PUBLIC_FOOTER_TEXT=
 # Required: Gatus API base
 GATUS_API_BASE=https://status.twin.sh/api/v1
 
+# Optional: credentials if Gatus is protected with security.basic
+GATUS_API_USERNAME=
+GATUS_API_PASSWORD=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GATUS_API_PASSWORD key should go before the GATUS_API_USERNAME key

(UnorderedKey)